modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -9,4 +9,91 @@
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_fermis"
+	inhand_icon_state = "plushie_fermis"
+	attack_verb_continuous = list("cuddles", "petpatts", "wigglepurrs")
+	attack_verb_simple = list("cuddle", "petpatt", "wigglepurr")
+	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/merowr.ogg' = 1)
+
+/obj/item/toy/plush/sechound
+	name = "Sechound plushie"
+	desc = "An adorable stuffed toy of a SecHound."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_securityk9"
+	inhand_icon_state = "plushie_securityk9"
+	attack_verb_continuous = list("beeps", "boops", "pings")
+	attack_verb_simple = list("beep", "boop", "ping")
+	squeak_override = list('sound/machines/beep.ogg' = 1)
+
+/obj/item/toy/plush/medihound
+	name = "Medihound plushie"
+	desc = "An adorable stuffed toy of a medihound."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_medihound"
+	inhand_icon_state = "plushie_medihound"
+	attack_verb_continuous = list("beeps", "boops", "pings")
+	attack_verb_simple = list("beep", "boop", "ping")
+	squeak_override = list('sound/machines/beep.ogg' = 1)
+
+
+/obj/item/toy/plush/scrubpuppy
+	name = "SecHound plushie"
+	desc = "An adorable stuffed toy of a scrubpuppy."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_scrubpuppy"
+	inhand_icon_state = "plushie_scrubpuppy"
+	attack_verb_continuous = list("beeps", "boops", "pings")
+	attack_verb_simple = list("beep", "boop", "ping")
+	squeak_override = list('sound/machines/beep.ogg' = 1)
+
+
+/obj/item/toy/plush/meddrake
+	name = "MediDrake Plushie"
+	desc = "An adorable stuffed toy of a Medidrake."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_meddrake"
+	inhand_icon_state = "plushie_meddrake"
+	attack_verb_continuous = list("beeps", "boops", "pings")
+	attack_verb_simple = list("beep", "boop", "ping")
+	squeak_override = list('sound/machines/beep.ogg' = 1)
+
+
+/obj/item/toy/plush/secdrake
+	name = "SecDrake Plushie"
+	desc = "An adorable stuffed toy of a Secdrake."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_secdrake"
+	inhand_icon_state = "plushie_secdrake"
+	attack_verb_continuous = list("beeps", "boops", "pings")
+	attack_verb_simple = list("beep", "boop", "ping")
+	squeak_override = list('sound/machines/beep.ogg' = 1)
+
+
+/obj/item/toy/plush/mammal/fox
+	name = "Fox plushie"
+	desc = "An adorable stuffed toy of a SecHound."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_fox"
+	inhand_icon_state = "plushie_fox"
+	attack_verb_continuous = list("beeps", "boops", "pings")
+	attack_verb_simple = list("beep", "boop", "ping")
```

Comment:
wrong verbs

Reply:
whew , thanks I wouldn't have noticed. All the typos and mistakes should (hopefully) be corrected now .